File: documentation/user/en/query/requirements/examples/facet/facet-summary-of-reference-simple.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		HierarchyWithin(
        			"categories",
        			AttributeEquals("code", "e-readers")
        		),
        		AttributeEquals("status", "ACTIVE"),
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("en"))
        	),
        	Require(
        		FacetSummaryOfReference("brand", Counts)
        	)
        )
	)
);

File: documentation/user/en/query/requirements/examples/facet/facet-summary-of-reference-simple.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					hierarchyWithin(
						"categories",
						attributeEquals("code", "e-readers")
					),
					attributeEquals("status", "ACTIVE"),
					entityLocaleEquals(Locale.forLanguageTag("en"))
				),
				require(
					facetSummaryOfReference("brand", COUNTS)
				)
			)
		);
	}
);
